vllm/v1/attention/backends/mamba_attn.py

```diff
@@ -116,7 +117,23 @@ def build(self,
         has_initial_states = None
         prep_initial_states = False
 
-        state_indices_tensor = common_attn_metadata.block_table_tensor[:, 0]
+        if self.kv_cache_spec.cache_strategy == "disabled":
+            # Always return just a single block per each request:
+            state_indices_tensor = common_attn_metadata.block_table_tensor[:, 0]
+        else:
+            # Return a tensor of shape (#requests, #blocks for longest request)
+            # filled in with cached and newly allocated blocks for each request
+            cache_block_size = self.kv_cache_spec.block_size
+            seq_lens_cpu = common_attn_metadata.seq_lens_cpu
+            block_table_bounds_cpu = (seq_lens_cpu + cache_block_size - 1) // cache_block_size
+            max_num_blocks = block_table_bounds_cpu.max()        
+            paged_kv_indices = common_attn_metadata.block_table_tensor[:, :max_num_blocks]
+            if self.kv_cache_spec.cache_strategy == "last":
+                # TODO: The "last" strategy is not fully implemented yet
+                # In the "last" strategy, the allocator puts 2 block in front
+                # For easiness of handling, we move them to be two last in list
```

Comment:
If the last strategy is not fully implemented, would it be better to have the first PR containing only a single strategy (e.g. "all") and focus on getting that working?